modules/mini-games/Race.lua

```diff
@@ -413,11 +414,24 @@ local kill_biters = Token.register(function(name)
     end
 end)
 
+--- Unstuck yourself
+Commands.new_command('unstuck', 'Unstuck your car from walls')
+:register(function(player)
+    local car = cars[player.name]
+    if not car or not car.valid then
+        player.print("Not in a car!")
+        return Commands.error
```

Comment:
this can be return Commands.error("Not in a car!")

Reply:
Ok.